### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by 4cq2**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,7 +135,7 @@
                         + "transform: translate(-50%, -50%)'>"
                         + "Uh oh! We couldn't fully download everything we needed :("
                         + "<br/> "
-                        + "We will try again shortly. In the mean time, check for problems with your Internet connection!"
+                        + "We will try again shortly. In the mean time, make sure to allow scripts from jitsi.net, or check for problems with your Internet connection!"
```
[the remark this answers is not in the document]
Comment (4cq2) [reply index.html:138]: please see the attached issue, yes its a valid addition.

**Comment by saghul** [comment]
I'm strongly -1 on this approach since we can't know for sure why the asset failed to be loaded.

Also, the difference in domains is deployment specific.

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (damencho) [reply index.html:138]: But in the issue you are mentioning meet.jit.si. This is a project that is being deployed in many environments where meet.jit.si is one and the domain is used there, but no on other places.

**Comment by damencho** [comment]
Yeah, the wording seems fine to me. The script detects that something was not downloaded correctly and warns the user about that. 
Why it was not downloaded is sometimes not able to detect ... it can be someone blocking it on the way not only a plugin ... it can be a transparent proxy or some network inspection device on the way ... the fact is the needed resources are missing and this is what the message is about.

**Comment by 4cq2** [comment]
the issue as you said is the underlying reason is unknown. ideally you should provide NO hints to what the problem might be, but thats not the case. you are providing this suggestion:

> check for problems with your Internet connection!

when adblocking users are going to get this message, even if their connection is perfect. so you should either remove the warning about bad connection, or add another stipulating about allowing scripts from `jitsi.net`.

**Comment by damencho** [comment]
> or add another stipulating about allowing scripts from `jitsi.net`.

We cannot add jitsi.net as this domain is specific to a deployment and has nothing to do with jitsi-meet for the open-source users and their deployments. 
By the way do you have a screenshot when you hit the problem? You should be seeing there the domain that is blocked, does it?

**Comment by 4cq2** [comment]
> We cannot add jitsi.net as this domain is specific to a deployment and has nothing to do with jitsi-meet for the open-source users and their deployments.

looks like the issue is here:

https://github.com/jitsi/jitsi-meet/blob/2d14990b9ebc261dfccd51da49b7c6211b11a3da/index.html#L195-L196

not sure how these are resolving to `jitsi.net`. if they are being hosted from `meet.jit.si`. at any rate, ANY message would be better than the current one. even something like:

> if are you ad blocking, please allow 3rd-party scripts when using Jitsi

this is vague enough to bypass your concerns, but still helpful enough to point ad blocking users in the right direction.

> By the way do you have a screenshot when you hit the problem? You should be seeing there the domain that is blocked, does it?

it does not show the blocked domain, only that my internet connection supposedly has a problem. that's the whole point of this pull request, is to add the blocked domain.

> block

**Comment by damencho** [comment]
> not sure how these are resolving to `jitsi.net`.
This is a configuration and my config can be using domain `example.com` where the message will show `jitsi.net`. 


> it does not show the blocked domain, only that my internet connection supposedly has a problem. that's the whole point of this pull request, is to add the blocked domain.
> 
> >

Click show more and you will see it. That gives enough details for anyone experiencing an issue to figure out what is missing and to check for resolution.

**Comment by 4cq2** [comment]
as you can see from the image, the page is almost empty. so why hide that URL behind a button? also the page refreshes every second, so you could have to copy and paste VERY quickly, otherwise you lose it. overall I think the design and/or message could be improved, but the rapid closing of the issue and PR seem to loudly communicate that you both feel the design is perfect as is.

**Comment by damencho** [comment]
> also the auto page refresh is pointless, as it again assumes the issue is with the users internet connection, rather than a purposeful blocking of third party scripts.

Yep, this page was created for this use case which is the more common one, and more people are hitting that then anything else. And hiding a technical stuff from normal users is normal. 

The refresh is with growing backoff timing. So the first one is two seconds, but later ones are between 2 and 60 seconds. 

It is that the changes proposed does not fit that's why the PR was closed. 
The existing wording express exactly what happen, some required resources were not able to be downloaded. And addresses the most common use case problem with the connectivity.

**Comment by 4cq2** [comment]
you seen to be unable or unwilling to respect the fact that ad blocking users use this service. if that's the case its fine, but you should at least be honest to yourself and others about that fact. because if YOU DID care about the experience of ad block users, you would want to offer some communication to that end. not only are you not doing so, but:

- you're actively hiding the URL
- you're forcing a page refresh every second, making the URL harder to capture
- you refuse to change the above two when confronted with the drawbacks of this approach
- you refuse to even add a few words to the HTML to help along any ad blocking users having trouble

so again, all of the above is your choice, but you'd be lying if you said you care about the experience of ad blocking users of Jitsi, because ever single comment on this issue and PR contradict that viewpoint.

**Comment by damencho** [comment]
Not that we do not care, I have already said which is the common use case here, anything else is with a low user impact so  not a high priority and we do not want to change the default experience for majority of the users for some corner cases. 

Any PRs are welcome where those strings can be customized so you can customize it on your deployment and mention the domains you want to customize there.

**Comment by 4cq2** [comment]
I wasted my time already doing a PR that I knew was never going to be approved, not going to do that again. 

if you were sincere about this offer, you never would have closed this PR. you would have given the feedback and left the PR open to give me a chance to make the suggested edits. the fact that the issue and PR were immediately closed is a strong signal that you just simply don't care about this issue and just want it to go away at any cost. again that's your choice, but at least be honest about it. just say "we don't care about the experience of ad block users, and we aren't going to make any changes to that end, even if its a matter of adding a few words to some HTML".

**Comment by damencho** [comment]
I closed it cause I explained several times, that the 'jitsi.net' is deployment specific and cannot be added as you wanted as it means nothing for the other deployments out there used by the community. 
And I also explained why this is not considered issue, as the text displayed expresses reality, resources have not been downloaded. And there is no way to detect why resources cannot be downloaded. Is ti blocked by browser extension, is it blocked by antivirus, was it blocked by network filtering system, was it blocked by internet provider filtering ... we cannot put all the reasons in there... but this is me, I re-opened the PR if someone else want to give an opinion.

**Comment by 4cq2** [comment]
it wont let me reopen this, but it should still be open. also if it helps, I just had a similar problem on this URL:

https://www.zocdoc.com/search?insurance_carrier=-1&insurance_plan=-1&search_query=urticaria&searchType=specialty&searchOriginator=SearchBar&referrerType=HomePage

and instead of a misleading message, I got this helpful message:

> Please enable JS and disable any ad blocker

**Comment by 4cq2** [comment]
asdf

**Pull request closed without merging**